gateway-ha/src/main/java/com/lyft/data/gateway/ha/persistence/JdbcConnectionManager.java

```diff
@@ -3,6 +3,7 @@
 import com.lyft.data.gateway.ha.config.DataStoreConfiguration;
 import com.lyft.data.gateway.ha.persistence.dao.QueryHistory;
 
+import com.sun.istack.Nullable;
```

Comment:
did you mean import javax.annotation.Nullable; ?

Reply:
fixed